pkg/services/ngalert/notifier/multiorg_alertmanager_test.go

```diff
@@ -298,6 +299,76 @@ func TestMultiOrgAlertmanager_AlertmanagerFor(t *testing.T) {
 	}
 }
 
+func TestMultiOrgAlertmanager_ActivateHistoricalConfiguration(t *testing.T) {
+	configStore := NewFakeConfigStore(t, map[int64]*models.AlertConfiguration{})
+	orgStore := &FakeOrgStore{
+		orgs: []int64{1, 2, 3},
+	}
+	tmpDir := t.TempDir()
+	defaultConfig := `{"template_files":null,"alertmanager_config":{"route":{"receiver":"grafana-default-email","group_by":["grafana_folder","alertname"]},"templates":null,"receivers":[{"name":"grafana-default-email","grafana_managed_receiver_configs":[{"uid":"","name":"email receiver","type":"email","disableResolveMessage":false,"settings":{"addresses":"\u003cexample@email.com\u003e"},"secureSettings":null}]}]}}`
+	cfg := &setting.Cfg{
+		DataPath:        tmpDir,
+		UnifiedAlerting: setting.UnifiedAlertingSettings{AlertmanagerConfigPollInterval: 3 * time.Minute, DefaultConfiguration: defaultConfig}, // do not poll in tests.
+	}
+	kvStore := NewFakeKVStore(t)
+	provStore := provisioning.NewFakeProvisioningStore()
+	secretsService := secretsManager.SetupTestService(t, fakes.NewFakeSecretsStore())
+	decryptFn := secretsService.GetDecryptedValue
+	reg := prometheus.NewPedanticRegistry()
+	m := metrics.NewNGAlert(reg)
+	mam, err := NewMultiOrgAlertmanager(cfg, configStore, orgStore, kvStore, provStore, decryptFn, m.GetMultiOrgAlertmanagerMetrics(), nil, log.New("testlogger"), secretsService)
+	require.NoError(t, err)
+	ctx := context.Background()
+
+	// Ensure that one Alertmanagers is created per org.
+	{
+		require.NoError(t, mam.LoadAndSyncAlertmanagersForOrgs(ctx))
+		require.Len(t, mam.alertmanagers, 3)
+	}
+
+	// First, let's confirm the default configs are active.
+	cfgs, err := mam.getLatestConfigs(ctx)
+	require.NoError(t, err)
+	require.Equal(t, defaultConfig, cfgs[1].AlertmanagerConfiguration)
+	require.Equal(t, defaultConfig, cfgs[2].AlertmanagerConfiguration)
+	// store id for later use.
```

Comment:
You can leave this as it is, but...
```suggestion
	// Store id for later use.
```